lib/slax_web/websockets/issue.ex

```diff
@@ -10,50 +10,38 @@ defmodule SlaxWeb.Issue do
   def handle_event(%{"bot_id" => bot_id}) when not is_nil(bot_id), do: nil
 
   def handle_event(%{"thread_ts" => ts, "channel" => channel, "text" => text, "type" => "message"}) do
-    with true <- String.match?(text, ~r{([\S-]+/)?([\S-]+)?(#[0-9]+)}),
-         {:ok, issue} <- load_issue(text) do
-      repo_and_issue =
-        Regex.replace(~r".*/repos/(\S+)/(\S+)/issues/(\d+)$", issue["url"], "\\1/\\2/\\3")
-
-      Slack.post_message_to_thread(%{
-        text:
-          "<#{issue["html_url"]}|#{repo_and_issue}>: #{issue["title"]} #{labels_for_issue(issue)}",
-        channel: channel,
-        thread_ts: ts
-      })
+    with issues when issues != [] <- Regex.scan(~r{([\w-]+/)?([\w-]+)?(#[0-9]+)}, text) do
+      reply =
+        issues
+        |> Enum.map(fn issue ->
+          issue
+          |> List.first()
+          |> load_issue()
+        end)
+        |> Enum.join("\n")
+
+      Slack.post_message_to_thread(%{text: reply, channel: channel, thread_ts: ts})
     else
-      false ->
+      [] ->
         nil
-
-      {:error, repo_and_issue} ->
-        Slack.post_message_to_thread(%{
-          text: "Issue #{repo_and_issue}: not found",
-          channel: channel,
-          thread_ts: ts
-        })
     end
   end
 
   def handle_event(%{"channel" => channel, "text" => text, "type" => "message"}) do
-    with true <- String.match?(text, ~r{([\S-]+/)?([\S-]+)?(#[0-9]+)}),
-         {:ok, issue} <- load_issue(text) do
-      repo_and_issue =
-        Regex.replace(~r".*/repos/(\S+)/(\S+)/issues/(\d+)$", issue["url"], "\\1/\\2/\\3")
-
-      Slack.post_message_to_channel(%{
-        text:
-          "<#{issue["html_url"]}|#{repo_and_issue}>: #{issue["title"]} #{labels_for_issue(issue)}",
-        channel_name: channel
-      })
+    with issues when issues != [] <- Regex.scan(~r{([\w-]+/)?([\w-]+)?(#[0-9]+)}, text) do
+      reply =
+        issues
```

Comment:
maybe this could be broken out 
```
  def def load_issues_from_scan(issues) do
    issues
    |> Enum.filter(&(Enum.any?(&1)))
    |> Enum.map(fn [issue] -> load_issue(issue) end)
    |> Enum.join("\n")
  end
```